### DISCUSSION TITLE
How to remove icons in middle of chart?

### DISCUSSION THREAD

**Discussion by wenmoose** [jaybo/deepzoom-blog#45] [Q&A] [answered]
I just discovered Deep Zoom and have used it for my current cruise in the San Juans.  My question is:   the chart display has several icons in the center of the chart--time, date, a circle with an attached arrow.  Also, there are icons on the left side or the chart.  How can those icons be removed so as not to interfere with the chart appearance?   Thanks for any information.   (Interestingly, I used the chart today to find the shortest route from Jones Island to Bellingham and currents at the time of travel.  Very helpful)

**Comment by jaybo** [comment]
The weather text at center left can be made transparent using the slider in the Layers dialog.
![image](https://github.com/user-attachments/assets/12e618f7-6304-4ff8-b62f-2ad1abad4abf)

The circle at the center has multiple purposes:
1. Shows the point at which the Lat/Lng at the top left refers.
2. Shows the point at which NOAA Weather predictions refer. 
3. When shown with an arrow, indicates the predicted wind direction and strength.

So I'm reluctant to hide this.  Anybody else want this to be made transparent along with the weather text?

**Comment by jaybo** [comment, marked as answer]
Disable `Settings.Weather.Enable weather display` to completely hide everything weather related.
www.deepzoom.com/settings
